_layouts/default.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="{{ page.lang | default: site.lang | default: 'en-US' }}" class="no-js">
+    
+    <head>
+
+        <title>{{ site.title }}</title>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <script async defer src="https://cdn.simpleanalytics.io/hello.js"></script>
+        <noscript><img src="https://api.simpleanalytics.io/hello.gif" alt=""></noscript>
+      
+        <!-- Twitter cards -->
```

Comment:
Did you check if Twitter card preview still works fine? E.g. https://cards-dev.twitter.com/validator (needs to be deployed to public site afaik)

Should result in tweets like this with image: https://twitter.com/oskarth/status/1192749849369882630?s=20

Reply:
Yes, I checked. Nothing has changed for posts.